fill/proto/interactive_filler.go

```diff
@@ -438,3 +463,16 @@ func makePrefix(s string, field *desc.FieldDescriptor, ancestor []string, ancest
 func isOneOfField(field *desc.FieldDescriptor) bool {
 	return field.GetOneOf() != nil
 }
+
+func readFileFromRelativePath(path string) ([]byte, error) {
+	if path == "" {
+		return nil, nil
+	}
+
+	absPath, err := filepath.Abs(path)
+	if err != nil {
+		return nil, err
+	}
```

Comment:
These lines are unnecessary because `ioutil.ReadFile` also accepts relative paths.

Reply:
didn't know that :)